### PR TITLE
Update Items_empty.txt

### PR DESCRIPTION
**Pull request by Faalagorn**
* Implemented suggestions from https://theindiestone.com/forums/index.php?/topic/24011-minor-correction-to-item-names-in-txt-files/ and #223
** Doesn't include book renaming, all of the consistent of/with naming container renaming and berries renaming from the post 
* Unified Bullet/Ammo/Round naming and changed .556 to 5.56
* T-Shirt → T-shirt, V-Neck → V-neck
* Various other minor fixes

Some items are singular some plural but that needs more rethinking as well. There are also two placeholder items (WaterDrop and something else I encountered; I didn't touch these.

### CODE REVIEW

File: Items_empty.txt [hunk 1]
```diff
@@ -223,8 +223,8 @@ Items_EN = {
 	DisplayName_Chipped_Stone = "Chipped Stone",
 	DisplayName_Chips = "Chips",
 	DisplayName_Chocolate = "Chocolate",
-	DisplayName_Choke_Tube_Full = "Choke Tube Full",
-	DisplayName_Choke_Tube_Improved = "Choke Tube Improved",
+	DisplayName_Choke_Tube_Full = "Full Choke Tube",
+	DisplayName_Choke_Tube_Improved = "Improved Choke Tube",
```
Comment (nsKpz) [inline Items_empty.txt:227]: While this name would make sense, the reason they're named like this is so they appear together when items in inventory are sorted alphabetically.

Comment (Faalagorn) [reply Items_empty.txt:227]: I thought about renaming items like these, some mods and translation already do it, but most of the items don't follow this rule, like paints, seeds or bullets.

Anyway, if you want to keep them that way, they should use some extra character, such as parenthesis or dash.

Comment (nsKpz) [reply Items_empty.txt:227]: Good point. We're gonna look into adding more display categories to items as renaming them all would indeed look odd at times.

Added dash to choke tube types, thank you!

**Comment by nsKpz** [comment]
Feedback from this was incorporated into the ItemName_EN.txt

**Pull request closed without merging**